app/controllers/api/resource_pools_controller.rb

```diff
@@ -3,5 +3,32 @@ class ResourcePoolsController < BaseController
     include Subcollections::Policies
     include Subcollections::PolicyProfiles
     include Subcollections::Tags
+
+  # GET /api/resource_pools
+  def index
+    if params[:type].present?
+      resource_pools = fetch_resource_pools_by_type(params[:type])
+      res = collection_filterer(resource_pools, :resource_pools, ResourcePool).flatten
+      counts = Api::QueryCounts.new(ResourcePool.count, res.count, resource_pools.count)
+      render_collection(:resource_pools, res, :counts => counts, :expand_resources => @req.expand?(:resources), :name => 'resource_pools')
+    else
+      super
+    end
+  end
+    
+    
+    private
+
+    # Fetch resource pools by exact type
+    def fetch_resource_pools_by_type(type)
```

Comment:
This is not the right way to do this.  I would expect a type parmeter incoming that lists the type you need, and then it's a normal STI query.